src/test/java/org/fundacionjala/coding/dennis/DigitalRootTest.java

```diff
@@ -0,0 +1,56 @@
+package org.fundacionjala.coding.dennis;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for Digital Root Kata.
+ */
+public class DigitalRootTest {
+
+    private int data;
+    private static final int ONE_DIGIT = 5;
+    private static final int IN_TWO_DIGITS = 43;
+    private static final int OUT_TWO_DIGITS_TEST = 7;
+    private static final int IN_TWO_DIGITS_REC = 96;
+    private static final int OUT_TWO_DIGITS_REC_TEST = 6;
+    private static final int IN_MORE_DIGITS = 493193;
+    private static final int OUT_MORE_DIGITS = 2;
```

Comment:
Don't use constants for this case, please declare them as local variables inside test method.